wolfcrypt/src/aes.c

```diff
@@ -2676,6 +2676,22 @@ static WARN_UNUSED_RESULT int wc_AesDecrypt(
             return BAD_FUNC_ARG;
         }
 
+#if !defined(WOLFSSL_AES_128)
+        if (keylen == 16) {
+          return BAD_FUNC_ARG;
```

Comment:
Indents should be 4 spaces.

Reply:
ah yes. I know that well. My bad for letting this one slip by. I've pushed an update to fix